package.json

```diff
@@ -1,21 +1,31 @@
 {
-  "name": "electrum-client",
-  "version": "0.0.7",
-  "description": "Electrum protocol client for node.js",
+  "name": "electrum-client-js",
+  "version": "0.1.0",
+  "description": "Electrum protocol client for JS",
```

Comment:
Technically it's for the browser :)

Reply:
It's for both browser and node.js.

Reply:
Ahhh. Can you add "browser and node.js" in the description. Some projects get lazy and JS doesn't always mean for Node.js and vice-versa :)